battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/model/GameMode.java

```diff
@@ -0,0 +1,11 @@
+package dreamteam.battleship.service.springcontroller.model;
+
+/**
+ * Created by ehsan on 09.08.16.
+ */
+public enum GameMode {
```

Comment:
Are names clear enough for others that this does NOT require docs? :book: are fun!